src/orient/rational-calc.orient

```diff
@@ -715,14 +735,23 @@ Filecoin:
       // TODO: missing disk access
       epost_cost = epost_inclusions_cost + epost_snark_cost
 
+Timing:
+  describe(malicious_regen_time, "Time for a malicious adversary to regenerate a sector", secs)
+  describe(time_amax, "Factor of how much slower than the blocktime the malicious encoding should take")
+
+Timing (TimingAssumption):
+  RegenAttack (StackedSDRParameters):
+  malicious_regen_time = (encoding_time_asic * drg_d) / (stacked_layers)
+  malicious_regen_time = block_time * time_amax
```

Comment:
Did you intend for these two lines to be indented so they would require the `StackedSDRParameters` flag?

Reply:
yes, fixed